abstract/common-abstract.md

```diff
@@ -1,4 +1,4 @@
-Distributed tracing is a set of tools and practices to monitor the health and reliability of a distributed application. A distributed application is an application that consists of multiple components that are deployed and operated separately. It is also known as micro-service.

```

Comment:
It seems most of this content is not relevant to baggage, there's really no need to mention distributed tracing.

Reply:
I agree with Yuri. Baggage could be used by any distrubuted system including one that is not traced at all.

Reply:
Thanks, I have removed this content related to distributed tracing, and made the abstract shorter per the discussion in the last WG meeting.